_topics/en/silent-payments.md

```diff
@@ -73,5 +73,29 @@ excerpt: >
 
 ---
 
+Traditionally, a user who receives payments should generate a new Bitcoin
+address for every payment. This is because receiving multiple payments
+to the same address reveals that the same user received those payments,
+even if the outputs are later spent in separate transactions.
+This is known as [address reuse][topic output linking].
+
+Using a new address often requires a secure interaction between sender
+and receiver so that the receiver can provide a fresh address every time.
+However, interaction is often infeasible and in many cases undesirable.
+
+With silent payments, a receiver can generate and publish a single silent
+payment address, eliminating the need for interaction.
+The sender then selects one or more of their chosen inputs and uses their
+secret key(s) together with public key of the silent payment address to
+derive a shared secret which is used to generate the destination.
+
+The intended recipient detects the payment by scanning transactions
+in the blockchain and performing an ECDH calculation with the summed
+input public keys of the transaction and the scan key from their address.
+The main downside is that it is more computationally expensive than
+simply scanning the UTXO set for a `scriptPubKey` as in [BIP32][] style wallets.
+Additionally, using silent payments in a collaborative setting is left for
```

Comment:
yes! upon closer inspection, I can see how this is confusing

thank you for explaining why this could be confusing for the readers, and offering concrete suggestions on how to improve clarity here 👍 love learning good stuff about better technical writing through this pr here!

pushed a commit to incorporate your feedback